packages/core/src/config/knapsack-pro-env.config.ts

```diff
@@ -1,10 +1,8 @@
-import childProcess = require('child_process');
+import { spawnSync, execSync } from 'child_process';
```

Comment:
Does this change require a high version of Node than we say in readme?

Reply:
Good question. Thanks for asking.

No, TypeScript complies to its configuration from tsconfig, so what you see in .ts files doesn't change what ends up in the .js files.

<img width="1677" alt="Screenshot 2024-12-06 at 14 43 17" src="https://github.com/user-attachments/assets/a6e03a2b-4df1-4296-b534-a2c788f5a604">